cai_causal_graph/identify_utils.py

```diff
@@ -0,0 +1,60 @@
+"""
+Copyright (c) 2023 by Impulse Innovations Ltd. Private and confidential. Part of the causaLens product.
+"""
+from typing import List
+
+from cai_causal_graph import CausalGraph
+from cai_causal_graph.exceptions import CausalGraphErrors
+
+
+def identify_confounders(graph: CausalGraph, treatment: str, outcome: str) -> List[str]:
```

Comment:
Confused by the note as it was missing a node: 

"
Note: The set of confounders is must be a sufficient adjustment set but not every adjustment set also constitutes a set of confounders. As example, consider this graph: `z -> u -> x ->` and `z -> y.` 

The confounder between `x` and `y` is clearly `z`, but `u` is actually also a sufficient (minimal) adjustment set. Since all causal effect estimators have the wording confounders = ... in their constructors, I believe that this is fine. Let me know if you believe otherwise.
"

The causal graph package should not care about effect estimation per say. I think the util function should give the correct answer for the graph. But I was not sure what the note was saying. Are we actually providing more nodes than the truth?

Reply:
Ah apologies, I was missing a `y` in the nodes.

All I was saying is that:
- A set of confounders is a sufficient adjustment set.
- An adjustment set, however, does not need to constitute a set of confounders.

I agree that we do not really care in the `cai-causal-graph` package, and I have modified the argument names to reflect that too.

Reply:
Updated the PR description as well.